hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/conf/HddsConfServlet.java

```diff
@@ -144,18 +143,18 @@ public BadFormatException(String msg) {
   }
 
   private void processConfigTagRequest(HttpServletRequest request, String cmd,
-      Writer out) throws IOException {
-    Gson gson = new Gson();
+                                       Writer out) throws IOException {
```

Comment:
nit: please avoid unnecessary whitespace change (also, such indentation triggers unnecessary changes, e.g. when method is renamed or visibility updated)

Reply:
Sorry about that!
Reverted the changes.